infra/templates/az-isolated-service-single-region/app.tf

```diff
@@ -0,0 +1,35 @@
+// This file contains all of the resources that exist within the app dev subscription. Design documentation
```

Comment:
Is this the file that's expected to grow as more and more resources need vnet integration?

Reply:
yes -- we'll be adding things like ACR to this file as they all fall within the app-dev subscription.